modules/form.js

```diff
@@ -50,6 +50,9 @@ itemForm.onsubmit = async (event) => {
   const quantity = document.getElementById('quantityInput').value;
   const price = document.getElementById('priceInput').value;
 
+  // now we need to add section (store section), quantityUnit
```

Comment:
Next PR will cover adding quantity unit - a string to indicate what the unit of measurement is - 1 lbs? 1 ounce? 1 bag? 

I need to think about how to handle things like... 1 14.5 ounce can - this is a valid thing to have a on a shopping list but we're not currently supporting it in any way.